### ISSUE TITLE
Seg-fault in LayerNormKernelImpl

### ISSUE THREAD

**Issue by HeikoSchuett** [pytorch/pytorch#28319]
## 🐛 Bug

While implementing a recurrent neural network with Layer normalization on the GPU I get a segmentation fault in the kernel implementation of the normalization. The model runs fine on CPU.

## To Reproduce

1. create a model with layerNormalization and try to run it on GPU
2. run forward pass

Error message in debugger is:

Thread 1 "python" received signal SIGSEGV, Segmentation fault.
0x00007fffbc6607e2 in at::native::(anonymous namespace)::LayerNormKernelImpl(at::Tensor const&, at::Tensor const&, at::Tensor const&, long, long, double, at::Tensor*, at::Tensor*, at::Tensor*) ()
   from [...]/lib/python3.7/site-packages/torch/lib/libtorch.so

## Expected behavior

not crash

## Environment
PyTorch version: 1.2.0
Is debug build: No
CUDA used to build PyTorch: 10.0.130

OS: Ubuntu 18.04.3 LTS
GCC version: (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
CMake version: Could not collect

Python version: 3.7
Is CUDA available: Yes
CUDA runtime version: Could not collect
GPU models and configuration: GPU 0: GeForce RTX 2080 Ti
Nvidia driver version: 418.87.00
cuDNN version: /usr/lib/x86_64-linux-gnu/libcudnn.so.7.6.4

Versions of relevant libraries:
[pip] numpy==1.17.2
[pip] numpydoc==0.9.1
[pip] torch==1.2.0
[pip] torchvision==0.4.0a0
[conda] _pytorch_select           0.2                       gpu_0  
[conda] blas                      1.0                         mkl  
[conda] mkl                       2019.4                      243  
[conda] mkl-service               2.3.0            py37he904b0f_0  
[conda] mkl_fft                   1.0.14           py37ha843d7b_0  
[conda] mkl_random                1.1.0            py37hd6b4f25_0  
[conda] pytorch                   1.2.0           cuda100py37h938c94c_0  
[conda] torchvision               0.4.0           cuda100py37hecfc37a_0

not sure why the script misses the cuda version: copied from nvidia-smi:
NVIDIA-SMI 418.87.00    Driver Version: 418.87.00    CUDA Version: 10.1

**Comment by t-vi** [comment]
Could you fill us in on some more details here?
Ideally you'd have a small runnable code bit that crashes PyTorch in the way you describe. If that isn't an option, shape information and the relevant parameters would be great.

**Comment by nikitaved** [comment]
@HeikoSchuett Could you please provide some small repro script? Then it would be easier to dive in directly... Thanks!
